src/api/controllers/users.controller.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import { Model } from "sequelize";
+
+import { USERS_NOT_FOUND, USER_NOT_FOUND } from "../../constants";
+
+import User from "../../types/user.type";
+import UserModel from "../../models/user.model";
+import UserService from "../../services/user.service";
+
+const userService = new UserService(UserModel);
+
+export const createUser = async (req: Request, res: Response) => {
+  const user: User = req.body;
+  
+  const createdUser: Model<User> = await userService.createUser(user);
+  res.send(createdUser);
+};
+
+export const getUsers = async (req: Request, res: Response) => {
+  const { loginSubstring = "", limit = 10 } = req.query;
```

Comment:
Consider moving any raw value (numeric/string etc) to a separate constant on top of the file or even somewhere centrally for the whole project.